theme/less/homepage.less

```diff
@@ -32,7 +52,6 @@ main article > *:first-child
   border-radius: 8px;
   border: 1px solid #dcdcdc;
   padding: 10px 10px 60px 10px;
-  margin: 20px 0 0 0;
 }
 
 .box img 
```

Comment:
- Icon margin-top needs to be set to 0
  - Use before
  - add class to icon
  - use important

Reply:
In the case of `:before`  we need to specify particular box in styles for each image via let's say `:nth-child()` selector which is not the best solution considering that count of boxes can be changed in future. Or we can give classes to boxes and use images with `:before`. Which is optimal solution in case or boxes as they are complex objects and having a class for each one i think is OK.

Reply:
In Drupal site the font-weight is 900, do I need to add one more font file, or using here 700 (for consistency sake) is OK , what you think?

Reply:
> classes to boxes and use images with :before

I think I have classes in mind when I was proposing, something along the lines `.icon .download`, `.icon-download`, i.e.: `<p class="icon download">`.

You can also put boxes outside of the `<article>`, that way content styles won't be applied to them and you will have all the freedom designing boxes.
```
<article>
  <h2></h2> 
  <img>
  <ol></ol>
</article>
<section class="box">
  ...
</section>
...
```

Reply:
> In Drupal site the font-weight is 900, do I need to add one more font file, or using here 700 (for consistency sake) is OK , what you think?

I don't have strong opinion, but I feel like Source Sans Pro's `400` is more closer to the Drupal's "normal" font, which is used on header, rather than Source Sans Pro's `600`, which is too bold in comparison. I can't say for sure if Source Sans Pro's `500` will fit here better, but that's only mediocre solution we can have with Source Sans Pro, though I'm not sure if that worth it as we will download whole font to show one sentence.

On a relevant note I did notice that Fonts you added are to divergent in sizes, which is bit suspicious - 600 15kb, 400 87kb, make sense to double check if 400 contains on latin.

Following up: If it's about adding another 15kb font, then maybe you can experiment with 500.

Reply:
The problem with this solution is that `<article>` is in template `.ejs` file. Maybe we should go with simplest option and use here `!important`?

Reply:
> The problem with this solution is that <article> is in template .ejs file

Right, I think it would be less problem after we address -> https://github.com/browser-automation/cba-website/issues/30 or you can move box properties right away and move `.box`-es outside of article as part of current PR.

> Maybe we should go with simplest option and use here !important

Fine by me.

Reply:
> 
> Following up: If it's about adding another 15kb font, then maybe it worth experiment with 500.

It turned out that the real size of latin 400 font is again 15kb, but the version of 500 does not exist in google fonts repository. And consequently the font generator does give nothing -> https://fonts.googleapis.com/css?family=source+sans+pro:500&subset=latin

Reply:
I see, seems like Source Sans Pro doesn't have 500 weight